Formula/flagd.rb

```diff
@@ -0,0 +1,45 @@
+class Flagd < Formula
+  desc "Feature flag daemon with a Unix philosophy"
+  homepage "https://github.com/open-feature/flagd"
+  url "https://github.com/open-feature/flagd.git", using: :git, tag: "v0.3.0", revision: "2ad206e635cc5b68ae42287b740ddfc86b0194ff"
+  license "Apache-2.0"
+  head "https://github.com/open-feature/flagd.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPRIVATE"] = "buf.build/gen/go"
+    ldflags = %W[
+      -s -w
+      -X main.version=#{version}
+      -X main.commit=#{Utils.git_head}
+      -X main.date=#{time.iso8601}
+    ]
+
+    system "go", "build", *std_go_args(ldflags:, output: bin/"flagd")
+    generate_completions_from_executable(bin / "flagd", "completion")
+  end
+
+  test do
+    port = free_port
+
+    begin
+      pid = fork do
+        exec bin/"flagd", "start", "-f",
+             "https://raw.githubusercontent.com/open-feature/flagd/main/config/samples/example_flags.json", "-p", "#{port}"
+      end
+      sleep 3
+
+      resolve_boolean_command = <<-BASH
+        curl -X POST "localhost:#{port}/schema.v1.Service/ResolveBoolean" -d '{"flagKey":"myBoolFlag","context":{}}' -H "Content-Type: application/json"
+      BASH
+
+      expected_output = { "value": true, "reason": 'STATIC', "variant": 'on' }.to_json
+
+      assert_match expected_output, shell_output(resolve_boolean_command)
```

Comment:
validating that daemon returns feature flags values: https://github.com/open-feature/flagd/blob/main/docs/usage/evaluation_examples.md